docs/api-reference/geo-layers/tile-layer.md

```diff
@@ -187,16 +187,35 @@ The maximum memory used for caching tiles. If this limit is supplied, `getTileDa
 - Default: `null`
 
 
-##### `refinementStrategy` (Enum, optional)
+##### `refinementStrategy` (String|Function, optional)
 
-How the tile layer refines the visibility of tiles. One of the following:
+How the tile layer refines the visibility of tiles. When zooming in and out, if the layer only shows tiles from the current zoom level, then the user may observe undesirable flashing while new data is loading. By setting `refinementStrategy` the layer can attempt to maintain visual continuity by displaying cached data from a different zoom level before data is available.
+
+This prop accepts one of the following:
 
 * `'best-available'`: If a tile in the current viewport is waiting for its data to load, use cached content from the closest zoom level to fill the empty space. This approach minimizes the visual flashing due to missing content.
 * `'no-overlap'`: Avoid showing overlapping tiles when backfilling with cached content. This is usually favorable when tiles do not have opaque backgrounds.
 * `'never'`: Do not display any tile that is not selected.
+* A custom function. See "custom strategy" below.
 
 - Default: `'best-available'`
 
+##### custom strategy
+
+Apps may defin a custom `refinementStrategy` by supplying its own callback function. The function will be called frequently on every viewport update and every tile loaded event.
+
+When called, the function receives an array of `Tile` instances representing every tile that is currently in the cache. Each tile is populated with the following properties:
+  - `x` (Number) - x index of the tile
+  - `y` (Number) - y index of the tile
+  - `z` (Number) - z index of the tile
+  - `parent` (Tile) - the parent tile (a tile on `z-1` that contains this tile), if present in the cache
```

Comment:
Can there sometimes be corner cases where we zoom quickly and some of the parents in the chain are not available? 

I suppose in this case we will not be able to prevent the image from "blanking", even if we implement this callback. 

That is probably OK for now.

Reply:
Looked at the code again - right now `parent` is set to the nearest ancestor, so not necessarily z+1.

Reply:
That could be useful. Maybe call it `nearestAncestor` then?